src/csrs/service/csrsService.ts

```diff
@@ -25,16 +47,22 @@ export class CsrsService {
 	}
 
 	async getDepartmentCodeToNameMapping() {
-		const organisations = JsonpathService.jsonpath().query(
-			await this.getOrganisations(),
-			'$._embedded.organisations.*'
-		)
-		const codeToName: any = {}
+		let departmentCodeToNameMapping = this.cacheService.cache.get(CsrsService.DEPARTMENT_CODE_TO_NAME_MAPPING)
```

Comment:
MAINTAIN - We should aspire work with objects rather than key-value pairs. Easier to change the contract or display (if need be) and refactor!